### PR TITLE
Logrotate file for centos7 and rights

### LINKED ISSUE #1536: Logrotate on CentOS7 need a change
Need to add 'su pf pf' at the start of the logrotate file

### PR DESCRIPTION
**Pull request by whitx**
# Description

Added a file for logrotate on centos7 and fixing right for this file after installation
# Impacts

logrotate on RHEL system
# Issue

fixes #1536 
# Delete branch after merge

NO
# NEWS file entries
## Enhancements
- Added a option tu run logrotate as the user pf on centos 7

### CODE REVIEW

File: addons/packages/packetfence.spec [hunk 1]
```diff
@@ -477,8 +477,12 @@ cp -r addons/watchdog/ $RPM_BUILD_ROOT/usr/local/pf/addons/
 cp -r addons/AD/* $RPM_BUILD_ROOT/usr/local/pf/addons/AD/
 cp addons/*.pl $RPM_BUILD_ROOT/usr/local/pf/addons/
 cp addons/*.sh $RPM_BUILD_ROOT/usr/local/pf/addons/
+%if 0%{?el7}
+cp addons/logrotate-c7 $RPM_BUILD_ROOT/usr/local/pf/addons/
```
Comment (jrouzierinverse) [inline addons/packages/packetfence.spec]:
```
mv -f addons/logrotate-c7 $RPM_BUILD_ROOT/usr/local/pf/addons/logrotate
```

File: addons/packages/packetfence.spec [hunk 2]
```diff
@@ -477,8 +477,12 @@ cp -r addons/watchdog/ $RPM_BUILD_ROOT/usr/local/pf/addons/
 cp -r addons/AD/* $RPM_BUILD_ROOT/usr/local/pf/addons/AD/
 cp addons/*.pl $RPM_BUILD_ROOT/usr/local/pf/addons/
 cp addons/*.sh $RPM_BUILD_ROOT/usr/local/pf/addons/
+%if 0%{?el7}
+mv -f addons/logrotate-c7 $RPM_BUILD_ROOT/usr/local/pf/addons/logrotate
```
Comment (fdurand) [inline addons/packages/packetfence.spec]: why not a cp like for centos 6 ?

Comment (jrouzierinverse) [inline addons/packages/packetfence.spec]: After looking this we should not do the copy here.

File: addons/packages/packetfence.spec [hunk 3]
```diff
@@ -477,8 +477,12 @@ cp -r addons/watchdog/ $RPM_BUILD_ROOT/usr/local/pf/addons/
...
+%else
 cp addons/logrotate $RPM_BUILD_ROOT/usr/local/pf/addons/
-%{__install} -D -m0755 addons/logrotate $RPM_BUILD_ROOT/etc/logrotate.d/packetfence
+%endif
+%{__install} -D -m0664 addons/logrotate $RPM_BUILD_ROOT/etc/logrotate.d/packetfence
```
Comment (jrouzierinverse) [inline addons/packages/packetfence.spec:483]: Here is the better place do make that decision of which logrotate to use.

```
%if 0%{?el7}
%{__install}  -m0664 addons/logrotate-c7 $RPM_BUILD_ROOT/etc/logrotate.d/packetfence/logrotate
%else
%{__install} -m0664 addons/logrotate $RPM_BUILD_ROOT/etc/logrotate.d/packetfence/logrotate
%endif
```

**Pull request merged**